conf/azure.yaml.template

```diff
@@ -5,6 +5,8 @@ AZURERM:
   CLIENT_SECRET:
   # The subscription ID is a GUID that uniquely identifies your subscription to use Azure services.
   SUBSCRIPTION_ID:
+  # The resource group where the resources will be created under above subscription
+  RESOURCE_GROUP: SatelliteQEResources
```

Comment:
The actual resource group name shouldn't be in the template.

Reply:
Done!